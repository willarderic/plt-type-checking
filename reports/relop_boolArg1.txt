TOKEN: LPAREN  LEXEME: (					TOKEN: LPAREN    LEXEME: (
TOKEN: NE  LEXEME: /=						TOKEN: NE        LEXEME: /=
REL_OP -> /=							REL_OP -> /=
BIN_OP -> REL_OP						BIN_OP -> REL_OP
TOKEN: T  LEXEME: t						TOKEN: T         LEXEME: t
CONST -> t							CONST -> t
EXPR -> CONST							EXPR -> CONST
TOKEN: INTCONST  LEXEME: 1					TOKEN: INTCONST  LEXEME: 1
CONST -> INTCONST						CONST -> INTCONST
EXPR -> CONST							EXPR -> CONST
ARITHLOGIC_EXPR -> BIN_OP EXPR EXPR				ARITHLOGIC_EXPR -> BIN_OP EXPR EXPR
Line 1: Arg 1 must be integer or string				Line 1: Arg 1 must be integer or string
