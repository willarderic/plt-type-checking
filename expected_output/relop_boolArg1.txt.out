TOKEN: LPAREN    LEXEME: (
TOKEN: NE        LEXEME: /=
REL_OP -> /=
BIN_OP -> REL_OP
TOKEN: T         LEXEME: t
CONST -> t
EXPR -> CONST
TOKEN: INTCONST  LEXEME: 1
CONST -> INTCONST
EXPR -> CONST
ARITHLOGIC_EXPR -> BIN_OP EXPR EXPR
Line 1: Arg 1 must be integer or string
